templates/set-lcm-mode.ps1.erb

```diff
@@ -0,0 +1,51 @@
+Write-Host "Setting LCM"
+$LCMRefreshMode = '<%= @refresh_mode %>'
+$generationDate = (get-date -format "M/d/yyyy HH:mm:ss")
+$encoding       = [system.Text.Encoding]::UTF8
+$Namespace      = 'root/Microsoft/Windows/DesiredStateConfiguration'
+$ClassName      = 'MSFT_DSCLocalConfigurationManager'
+
+Write-Verbose "LCM Mode: $LCMRefreshMode"
```

Comment:
So FYI I don't think we have a way to pass in a `-Verbose` to PowerShell from Ruby...